src/Utilities.UnitTests/ToolTask_Tests.cs

```diff
@@ -780,5 +780,19 @@ public void GetProcessStartInfoCanOverrideEnvironmentVariables()
             Assert.Equal(false, task.StartInfo.EnvironmentVariables.ContainsKey("a"));
 #endif
         }
+
+        [Fact]
+        public void VisualBasicLikeEscapedQuotesInCommandAreNotMadeForwardSlashes()
+        {
+            MyTool t = new MyTool();
+            MockEngine engine = new MockEngine();
+            t.BuildEngine = engine;
+            t.MockCommandLineCommands = NativeMethodsShared.IsWindows
+                                            ? "/C echo \"hello \\\"world\\\"\""
```

Comment:
Since quotes are special in verbatim strings it's still confusing. The question is whether you prefer

```
 "-c echo \"hello \\\"world\\\"\""
```

or

```
@"-c echo ""hello \""world\"""""
```

In my head the former is marginally less confusing. But juuuuuuuust barely!